filebeat/prospector/prospector.go

```diff
@@ -38,15 +38,15 @@ type Prospectorer interface {
 	Run()
 }
 
-type Outlet interface {
-	OnEvent(event *input.Event) bool
+type Output interface {
+	Send(event *input.Event) bool
 }
```

Comment:
this is not necessarily `Output`, but more like an `Eventer` with `OnEvent` method?